<ui version="4.0" >
 <class>NormalizeDlg</class>
 <widget class="QDialog" name="NormalizeDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Normalize</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTextBrowser" name="m_pOutM" >
     <property name="tabChangesFocus" >
      <bool>true</bool>
     </property>
     <property name="lineWrapMode" >
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_pDetailE" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pKeepOpenCkM" >
        <property name="text" >
         <string>Keep window open after completion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pAbortB" >
        <property name="text" >
         <string>Abort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pCloseB" >
        <property name="text" >
         <string>Close</string>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pDetailE</tabstop>
  <tabstop>m_pKeepOpenCkM</tabstop>
  <tabstop>m_pAbortB</tabstop>
  <tabstop>m_pCloseB</tabstop>
  <tabstop>m_pOutM</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
